### DISCUSSION TITLE
Visual Bug, while searching recipes

### DISCUSSION THREAD

**Discussion by TruePitbull** [shedaniel/RoughlyEnoughItems#1756] [Q & A]
When i hover over a Item, to show the recipe, I'll have this bug.

https://github.com/user-attachments/assets/64a1cdcb-11f0-4c64-9769-19d86460b418

**Comment by SirGoldalot** [comment]
BUMP
this is happening to me as well and have confirmed by removing all other mods and this issue still happens

**Comment by Fyoncle** [comment]
BUMP
